auth/api/http/groups/transport.go

```diff
@@ -327,19 +326,22 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 		w.WriteHeader(http.StatusForbidden)
 	case errors.Contains(err, auth.ErrMemberAlreadyAssigned):
 		w.WriteHeader(http.StatusConflict)
-	case errors.Contains(err, io.EOF):
-		w.WriteHeader(http.StatusBadRequest)
-	case errors.Contains(err, io.ErrUnexpectedEOF):
-		w.WriteHeader(http.StatusBadRequest)
```

Comment:
Please doublecheck. These errors are not returned by the service layer but can happen during the decoding of the request body.

Reply:
This errors are related to protobuf Unmarshal. If there is any error returned we should Wrap it with `ErrMalformedEntity`.  If it occurs in a consumer handler then tthe error is simply never returned.